erc20-connector/contracts/ProofKeeper.sol

```diff
@@ -0,0 +1,56 @@
+pragma solidity ^0.6.12;
+
+import "rainbow-bridge/contracts/eth/nearprover/contracts/INearProver.sol";
+import "rainbow-bridge/contracts/eth/nearprover/contracts/ProofDecoder.sol";
+import "rainbow-bridge/contracts/eth/nearbridge/contracts/Borsh.sol";
+
+contract ProofKeeper {
+    using Borsh for Borsh.Data;
+    using ProofDecoder for Borsh.Data;
+
+    INearProver public prover_;
+    bytes public nearProofProducerAccount_;
```

Comment:
Same with this name. While you are trying to use generic name, `nearTokenFactory` conveys better the idea of what is happening here IMO.

Reply:
The `ERC20Locker` passes the `nearTokenFactory` as an argument to this contract, so I think it already specifies how this will be used. Nevertheless, I'm ok to keep the old name if you think of this contract as specific to this connector.